LoopKit/InsulinKit/DoseStore.swift

```diff
@@ -831,6 +830,105 @@ extension DoseStore {
         }
     }
 
+    /**
+     Adds and persists external dose entries
+     - parameter doses: An array of dose entries to add.
+     - parameter completion: A closure called after the doses are saved. The closure takes a single argument:
+     - parameter error: An error object explaining why the doses could not be saved.
+     */
+    public func logOutsideDose(_ doses: [DoseEntry], completion: @escaping (_ error: Error?) -> Void) {
```

Comment:
[nit] `logOutsideDose` implies to me a _logging_ operation, not a _storage_ operation.  Consider renaming `storeOutsideDose`.

Reply:
See refactoring for "manuallyEntered"